packages/gatsby-source-wordpress/src/steps/ingest-remote-schema/identify-and-store-ingestable-types.js

```diff
@@ -38,53 +44,15 @@ const identifyAndStoreIngestableFieldsAndTypes = async () => {
         const lazyType = typeMap.get(typeName)
         store.dispatch.remoteSchema.addFetchedType(lazyType)
       }
-    })
-  }
-
-  const interfaces = introspectionData.__schema.types.filter(
-    type => type.kind === `INTERFACE`
-  )
-
-  for (const interfaceType of interfaces) {
-    if (typeIsExcluded({ pluginOptions, typeName: interfaceType.name })) {
-      continue
-    }
-
-    if (!interfaceType.fields) {
-      continue
-    }
-
-    const typesThatImplementInterface =
-      getTypesThatImplementInterfaceType(interfaceType)
 
-    const shouldSkipInterfaceType = !transformFields({
-      fields: interfaceType.fields,
-      parentType: interfaceType,
-      parentInterfacesImplementingTypes: typesThatImplementInterface,
-    })
-
-    if (shouldSkipInterfaceType && interfaceType.name !== `Node`) {
-      continue
-    }
-
-    store.dispatch.remoteSchema.addFetchedType(interfaceType)
-
-    if (interfaceType.fields) {
-      for (const interfaceField of interfaceType.fields) {
-        if (interfaceField.type) {
-          store.dispatch.remoteSchema.addFetchedType(interfaceField.type)
-        }
+      if (typeSettings.nodeInterface) {
+        nodeInterfaceTypes.push(typeName)
       }
```

Comment:
I moved this lower in the file so that interface types are inspected for possibly being pulled into Gatsby after we have info about which types will be node types in Gatsby